x-pack/plugins/ml/public/application/components/scatterplot_matrix/scatterplot_matrix.tsx

```diff
@@ -138,22 +144,26 @@ export const ScatterplotMatrix: FC<ScatterplotMatrixProps> = ({
 
     async function fetchSplom(options: { didCancel: boolean }) {
       setIsLoading(true);
+      const messages: string[] = [];
+
       try {
+        const outlierScoreField = `${resultsField}.${OUTLIER_SCORE_FIELD}`;
+        const includeOutlierScoreField = resultsField !== undefined;
+
         const queryFields = [
           ...fields,
           ...(color !== undefined ? [color] : []),
-          ...(legendType !== undefined ? [] : [`${resultsField}.${OUTLIER_SCORE_FIELD}`]),
+          ...(includeOutlierScoreField ? [outlierScoreField] : []),
         ];
 
-        const queryFallback = searchQuery !== undefined ? searchQuery : { match_all: {} };
```

Comment:
Will the searchQuery passed in already be the default one when no query is set?

Reply:
Yes, I noticed this during updating this PR that the fallback to `match_all` here is unnecessary.